src/styles/components/_component-card.scss

```diff
@@ -48,9 +48,7 @@
       z-index: 1;
       font-size: 1.125rem; // 18px/16
       line-height: 4.6667rem; // 75px
-      letter-spacing: 0.02em;
-      border-top: 0.25rem solid var(--foreground-color);
-      background-color: var(--background-color);
+      background-color: var(--foreground-color);
```

Comment:
I think we want `var(--background-color)` here, then `var(--foreground-color)` down below in the media query for dark mode. We want to avoid looking like this in light mode:

![Screenshot 2022-11-09 at 3 47 30 PM](https://user-images.githubusercontent.com/82047862/200960001-81fb7834-48a6-44fa-9b42-3def1043160f.png)

Reply:
Fixed this too :)

![image](https://user-images.githubusercontent.com/49820744/200977192-b1ead7a1-80b5-428f-a6a3-bddc0eb5a6b9.png)